### PR TITLE
add docs for length and less than task

### PR DESCRIPTION
**Pull request by infiloop2**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.Any change needs to be discussed before proceeding.**

## Closing issues

Not any issue, adding docs for new tasks added in https://github.com/smartcontractkit/chainlink/pull/7423/files

...

## Description

Adding documentation for new tasks added in https://github.com/smartcontractkit/chainlink/pull/7423/files

...

## Changes

add docs for two new tasks

### CODE REVIEW

**Comment by dwightjl** [comment]
Thank you for the PR! Pointing this to the cl-next branch until https://github.com/smartcontractkit/chainlink/pull/7423/ is included in a release branch.

File: docs/chainlink-nodes/oracle-jobs/task-types/task_length.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: nodes.liquid
+section: nodeOperator
+date: Last Modified
+title: "Length Task"
+permalink: "docs/jobs/task-types/length/"
+---
+
+Returns the length of a byte array or string.
```
Comment (pinebit) [inline docs/chainlink-nodes/oracle-jobs/task-types/task_length.md:9]: Maybe we need to be clear on the behavior for unicode characters in a string. Does it return the number of runes in this case?

**Review by pinebit — APPROVED** [review]

**Review by pinebit — COMMENTED** [review]
Need to add `gasUnlimited` param description here or in another PR.

File: docs/chainlink-nodes/oracle-jobs/task-types/task_eth_call.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ Makes a non-mutating contract call to the specified contract with the specified
 - `contract`: the address of the contract to call.
 - `data`: the data to attach to the call (including the function selector).
 - `gas`: the amount of gas to attach to the transaction.
+- `unlimited_gas`: boolean indicating if unlimited gas should be provided for the call. If set to true, `gas` parameter should not be passed.
```
Comment (jmank88) [inline docs/chainlink-nodes/oracle-jobs/task-types/task_eth_call.md:16]: s/unlimited_gas/gasUnlimited?
```suggestion
- `gasUnlimited`: boolean indicating if unlimited gas should be provided for the call. If set to true, `gas` parameter should not be passed.
```

Comment (FelixFan1992) [reply docs/chainlink-nodes/oracle-jobs/task-types/task_eth_call.md]: Done

**Pull request merged**